apps/aecore/test/aec_peers_pool_tests.erl

```diff
@@ -1884,6 +1910,9 @@ rand_state() ->
 
 -endif.
 
+make_env(Opts) ->
```

Comment:
So you overwrite a "global" variable many times over... I wonder if this is going to disturb system test or the simulator, in which one would like this option to play around.

Reply:
It would be much more natural to have the original Argument to new and get a list of options, possibly read from the environment if not coded in the application?

Reply:
I could create a function `aec_peers_pool:config` which would create the `Opts` which then could be passed to `new/1`?

Reply:
I think this is solved, the `new/1` is still there